### PR TITLE
add aarch64/arm64 docker build

### PR DESCRIPTION
**Pull request by Zoey2936**
Signed-off-by: Zoey <zoey@z0ey.de>
Needed for https://github.com/nextcloud/all-in-one/pull/1583

### CODE REVIEW

**Comment by Zoey2936** [comment]
@fancycode you seem to be the most active person in this repo, could you please have a look on this PR? Thanks!

**Comment by fancycode** [comment]
Yeah, sorry for the delay - I'm still catching up after the holidays. I'll try to review it this week.

**Comment by Zoey2936** [comment]
> Yeah, sorry for the delay - I'm still catching up after the holidays. I'll try to review it this week.

Many thanks, take the time you need.

**Comment by fancycode** [comment]
Thanks!

File: docker/proxy/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
-FROM golang:1.19 AS builder
+FROM --platform=${BUILDPLATFORM} golang:1.19 AS builder
```
Comment (swixfort) [inline docker/proxy/Dockerfile:1]: Hi @Zoey2936,
I happened to test the docker(-compose) version atm and realized that the initial build did fail for 2 reasons. One of which is the use of args here and the implication that buildkit is being used. It's not mentioned in the docs though (as far as I have seen).
I'm not sure if this commit should've included a note in the [README.md](https://github.com/strukturag/nextcloud-spreed-signaling#running-with-docker) in the Docker section.

Comment (Zoey2936) [reply docker/proxy/Dockerfile:1]: that running the compose file requires builds or what exactly?

Comment (fancycode) [reply docker/proxy/Dockerfile:1]: I just added #408 to fix CI and mention that docker-compose v2 is required for building.

See https://github.com/strukturag/nextcloud-spreed-signaling/commit/2582e4ffb44de622b66f21007ee784161c9c7dd1 for discussion.

Comment (fancycode) [reply docker/proxy/Dockerfile:1]: @swixfort did the build succeed with a newer version of docker-compose?

Comment (swixfort) [reply docker/proxy/Dockerfile:1]: I need to check on a different machine because I did manage to alter the config for my setup earlier.

Comment (swixfort) [reply docker/proxy/Dockerfile:1]: @fancycode I did successfully compose with v2.

@Zoey2936 basically, though with docker-compose v2 and the addition from @fancycode it's solved now :-)

Comment (fancycode) [reply docker/proxy/Dockerfile:1]: Great, thanks for confirming.

**Comment by Zoey2936** [comment]
@fancycode I know it's generally not supposed to ask for release dates, but can you maybe say roughly when a new stable release with a multiarch docker image will be released? Or would it be possible for you to rebuild the latest release (1.0.0), so it would be available with multiarch support? Many Thanks!

**Comment by fancycode** [comment]
I have a new release planned for next week, more or less when#420 is resolved.

**Comment by Zoey2936** [comment]
Thanks!

**Pull request merged**